jalib/jassert.cpp

```diff
@@ -160,7 +174,13 @@ _open_log_safe(const jalib::string &s, int protectedFd)
 }
 
 static jalib::string&
-theLogFilePath() { static jalib::string s; return s; }
+theLogFilePath() {
+  static jalib::string *s = NULL;
+  if (s == NULL) {
```

Comment:
Suggest adding comment:
// Technically, this is a memory leak, but `s` is static and so it happens only once.